edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -88,6 +92,36 @@ impl DockerModuleRuntime {
         self
     }
 
+    pub fn with_network_configuration(mut self, network_configuration: MobyNetwork) -> Self {
+        self.network_id = Some(network_configuration.name().to_string());
+        if let MobyNetwork::Network(network) = network_configuration {
+            self.ipv6 = network.ipv6().unwrap_or_else(|| &false).to_owned();
+            if let Some(ipam_config) = network.ipam() {
+                let mut config: Vec<::std::collections::HashMap<String, String>> = Vec::new();
```

Comment:
Build the Vec from the iterator rather than pushing into it manually.

`let config: Vec<_> = ipam_config.iter().map(|ipam_config| { ... }).collect();`